clusterloader2/testing/experimental/storage/pod-startup/config.yaml

```diff
@@ -7,12 +7,22 @@
   {{$NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 100}}
 
 # Test Variales
-  {{$PODS_PER_NODE := .PODS_PER_NODE}}
+  {{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 1}}
   {{$DEPLOYMENT_TEMPLATE_PATH := .DEPLOYMENT_TEMPLATE_PATH }}
-  {{$VOLUMES_PER_POD := .VOLUMES_PER_POD}}
+  {{$VOLUMES_PER_POD := DefaultParam .VOLUMES_PER_POD 1}}
   {{$VOLUME_TEMPLATE_PATH := .VOLUME_TEMPLATE_PATH}}
+  # Set this to false if apiserver, kube-controller-manager and kube-scheduler
```

Comment:
But you do for the other components?

I need to double-check whether I can drop the "insecure-port: '8080'" in my local cluster config. It might be that I added it pro-actively.